spring-cloud-stream/src/main/java/org/springframework/cloud/stream/annotation/StreamConverter.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2017 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.stream.annotation;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import org.springframework.beans.factory.annotation.Qualifier;
+
+/**
+ * Marker to tag {@link org.springframework.messaging.converter.MessageConverter} beans that will be added to the {@link org.springframework.cloud.stream.converter.CompositeMessageConverterFactory}
+ * @author Vinicius Carvalho
+ */
+@Target({ElementType.FIELD,ElementType.METHOD})
+@Retention(RetentionPolicy.RUNTIME)
+@Documented
+@Qualifier
+public @interface StreamConverter {
+}
```

Comment:
Are we settling on this name? Without reading javadocs, it is not apparent to me what this annotation is used for. Any possibilities to reflect what the annotation is intended for from the name itself? `StreamConverter` could mean a few things in the context of stream processing. We may want to convey what it is from the name itself since it feels like a fundamental part of the core stream API.

Reply:
I'm open to suggestions, the name came from a chat with @artembilan. I have no strong opinions on it.

Reply:
Allow me to suggest @StreamMessageConverter. The word "message" helps disambiguate and narrow down its purpose. M

Reply:
This has been named after `@IntegrationConverter` - just a blast after quick brainstorm.

I like @PedroAlvarado 's `@StreamMessageConverter`!